### DISCUSSION TITLE
Small up-shift in the x-y plane after a 6-min EPI scan.

### DISCUSSION THREAD

**Discussion by DuohuaSun** [pulseq/pulseq#93] [Q&A]
We noticed that there is a slightly up-shift (phase-encoding direction) in the x-y plane after a 6-min EPI scan, no such issue founded in GRE. We've tried both version 1.3.1 and version 1.4.2, this upper shifting problem still exist. Any solutions?

Thank you.

**Comment by m-a-x-i-m-z** [comment]
Hi! 

it is a known generic problem of EPI and is corrected in the product sequence in the image reconstruction. The physical source of the problem is the drift of the center frequency of the MR system, which mainly occurs due to the change of the temperature of the shimming iron. That iron is deposited in specialized pockets inside of your gradient coil when the scanner is installed to homogenize the main magnetic field (so-called static shimming). 

If your EPI sequence has the 3-echo "navigator  scan" (also known as phase correction scan) you can use the phase difference between the first and the third echo to detect frequency changes and correct for the frequency drift by applying a corresponding linear phase modulation along the ky direction prior to the FFT. 

Hope this helps! 

Best regards, 
Maxim & the Pulseq team

**Comment by DuohuaSun** [comment]
Hi Maxim,

Thank you so much for your reply, very helpful! It’s good to know the reason that causes the problem. We will try the solution you mentioned.

Best regards,
Duohua.
